changes/unreleased/Changed-20220304-221506.yaml

```diff
@@ -0,0 +1,5 @@
+kind: Changed
+body: Move to operator-sdk v1.18.0
+time: 2022-03-04T22:15:06.6959129+01:00
+custom:
+  Issue: "30"
```

Comment:
We should use 165 as the issue number.  I typically try to link with the PR that pushed the change as that gives more context.